src/material-experimental/mdc-slider/slider.ts

```diff
@@ -22,13 +75,229 @@ import {SliderAdapter} from './slider-adapter';
   selector: 'mat-slider',
   templateUrl: 'slider.html',
   styleUrls: ['slider.css'],
+  host: {
+    'class': 'mat-mdc-slider mdc-slider',
+    '[class.mdc-slider--range]': 'isRange',
+    '[class.mdc-slider--disabled]': 'isDisabled',
+    '[class.mdc-slider--discrete]': 'isDiscrete',
+    '[class.mdc-slider--tick-marks]': 'hasTickMarks',
+  },
+  exportAs: 'matSlider',
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None,
 })
-export class MatSlider {
+export class MatSlider implements AfterViewInit, OnDestroy {
+  /** The slider thumb(s). */
+  @ViewChildren('thumb') _thumbs: QueryList<ElementRef<HTMLElement>>;
 
-  /** Instance of the MDC slider foundation for this slider. */
+  /** The slider thumb knob(s) */
+  @ViewChildren('knob') _knobs: QueryList<ElementRef<HTMLElement>>;
+
+  /** The active section of the slider track. */
+  @ViewChild('trackActive') _trackActive: ElementRef<HTMLElement>;
+
+  /** The sliders hidden range input(s). */
+  @ContentChildren(MatSliderThumb, { descendants: false }) _inputs: QueryList<MatSliderThumb>;
+
+  /** Whether the slider is disabled. */
+  @Input()
+  get isDisabled(): boolean { return this._isDisabled; }
```

Comment:
```suggestion
  get disabled(): boolean { return this._isDisabled; }
```
Just `disabled`, which matches the native API and all the other components (similar for other boolean properties)